common/config/presets/tari_config_example.toml

```diff
@@ -136,7 +139,8 @@ base_node_query_timeout = 120
 # (options: "DirectOnly", "StoreAndForwardOnly", DirectAndStoreAndForward". default: "DirectAndStoreAndForward").
 #transaction_routing_mechanism = "DirectAndStoreAndForward"
 
-scan_for_utxo_interval=60
+# UTXO scanning service interval (default = 12 hours, i.e. 60 * 60 * 12 seconds)
```

Comment:
Did this sneak in?

Reply:
That setting did not have a comment, and the spacing was not consistent, so it was fixed. No functional change.

Reply:
my bad, sorry